app/components/set-lightning-address-modal/set-lightning-address-modal.tsx

```diff
@@ -99,6 +100,12 @@ export const SetLightningAddressModal = ({
       return
     }
 
+    // Short delay to ensure GraphQL updates propagate before refreshing the current profile
+    await new Promise<void>((resolve) => {
+      setTimeout(() => resolve(), 100)
+    })
+
+    await updateCurrentProfile()
```

Comment:
this does not makes sense, you are adding the timeout before the update?  with what but is this related?